Formula/tbl2asn.rb

```diff
@@ -31,14 +32,15 @@ class Tbl2asn < Formula
   end
 
   def install
-    if OS.mac?
-      bin.install "mac.tbl2asn" => "tbl2asn"
-    elsif OS.linux?
-      bin.install "linux64.tbl2asn" => "tbl2asn"
+    bin.install Dir["tbl2asn*"].first => "tbl2asn"
+    if OS.linux?
       system "patchelf",
         "--set-interpreter", HOMEBREW_PREFIX/"lib/ld.so",
         "--set-rpath", HOMEBREW_PREFIX/"lib",
         bin/"tbl2asn"
+        # Normally we would use patchelf to make this change but it seems
+        # broken for this use-case. Use the Stick of Correction instead.
+        inreplace bin/"tbl2asn", "libidn.so.11", "libidn.so.12"
```

Comment:
😮 Wow. Amazed this works. I'll tuck that away in my back pocket.
Running `patchelf` a second time with `--replace-needed` may also work.

Reply:
Nope, tried that. I think there's a bug in patchelf with respect to how `tbl2asn` is linked.

Reply:
The old binary patch! Nice one. 

The last time I did that was to patch the Roche 454 Newbler assembler to use our Ion Torrent mate pair linker sequence instead.

Reply:
Amazing! That's a good story!